### ISSUE TITLE
[BUG] crc on Windows waits for an IP forever

### ISSUE THREAD

**Issue by gbraad** [code-ready/crc#1603]
Start hangs on
```
INFO Creating CodeReady Containers VM for OpenShift 4.5.14...
```
when the network interface does not get an IP address assigned.
![image](https://user-images.githubusercontent.com/1894/96556543-ad5b7a00-12eb-11eb-8790-8a38a1ef0009.png)

**Comment by gbraad** [comment]
needs a timeout or will be solved with the vsock implementation

**Comment by gbraad** [comment]
forgot to modify the title ;-) was too focused on the image I guess

**Comment by daFritz84** [comment]
Any known workaround? I believe that I'm stuck on this issue getting crc up and running. 

From the VM Console, I can see that eth0 gets an external IP address assigned via DHCP, but for some reason, the Hypervisor does not show an IP Address like in the screenshot above.

**Comment by daFritz84** [comment]
@guillaumerose Hi, my issue has been resolved a while ago (see #1674 ). Unfortunately, I lack the environment to reproduce this problem now, so my test will not provide any new insights.
